{"name": "创建英雄失败的测试用例-攻击力不符合要求", "status": "passed", "parameters": [{"name": "success_for_name", "value": "'jinx'"}, {"name": "success_for_volume", "value": "1"}, {"name": "fail_for_power", "value": "0"}], "start": 1684150562963, "stop": 1684150562963, "uuid": "82e6780e-2e3e-4c71-8eca-69060c67a2d0", "historyId": "8a6c4314836fd09154cc3b6b96f95cd6", "testCaseId": "052da0c40443119f715fb4201d430dc8", "fullName": "test_create_hero_homework_01.TestHeroCreate#test_create_hero_fail_for_power", "labels": [{"name": "tag", "value": "run(order=4)"}, {"name": "suite", "value": "test_create_hero_homework_01"}, {"name": "subSuite", "value": "TestHeroCreate"}, {"name": "host", "value": "LAPTOP-0EHG7NSH"}, {"name": "thread", "value": "48660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_hero_homework_01"}]}